src/Microsoft.DotNet.Build.Tasks/PackageFiles/IL.targets

```diff
@@ -5,12 +5,13 @@
   <Target Name="CreateManifestResourceNames" Condition="'@(EmbeddedResource)' != ''" />
 
   <Target Name="GetIlasmPath"
-          Condition="'$(BuildingInsideVisualStudio)' == 'true'">
+          Condition="'$(OS)' == 'Windows_NT' And '$(IlasmToolPath)' == ''">
```

Comment:
I think I wold prefer we just remove this default ans always set the Path instead of having the OS condition, as I would like to be rid of those conditions.